documentation/performance/sizing-search.html

```diff
@@ -1,705 +1,477 @@
-</p><p>
-The target system utilization should be kept sufficiently low for the response
-times to be reasonable and within SLA even with some extra traffic occurring at peak hours.
+For example
+if we through <a href="#metrics-for-capacity-plan">metrics</a> see that the <em>DQW</em> = 0.50, the maximum speedup we can get by increasing parallelism by using more nodes and decreasing <em>DQW</em> is
+2. With fixed number of users, clients or load, <a href="https://en.wikipedia.org/wiki/Little%27s_law">Little's Law'</a> we can achieve 2x throughput if we are able to speed up latency by 2x.  
+<p><!-- depends on mathjax -->
+  $$\frac{1}{1 - \frac{0.5}{0.5+0.5}} = 2$$
 </p>
-
-
-
-<h2 id="data-volume-and-throughput-scaling-by-partitioning">Data volume and throughput scaling by partitioning</h2>
 <p>
-Each search/content node holds a partition of the index.
-Each search/content node queries its partition of the index upon a request from a <em>container</em>.
-A container is responsible for sending an incoming query in parallel
-to a set of search/content nodes so that all partitions are asked (full coverage).
-The results are then merged, and a complete result from the entire cluster is generated.
-The time to search the index is dependent on the number of nodes in the content cluster,
-since this determines the number of documents per node,
-which directly relates to <em>DQC</em> in the formulas above.
-The Vespa search kernel has performance characteristics where, within certain boundaries,
-the latency can be considered to scale linearly with the number of documents indexed on each node.
-See the graph below, where we have plotted the 99 percentile latency of a fixed set of
-queries executed against an increasing number of documents indexed.
+When <em>SQW</em> is no longer significantly less than <em>DQW</em>,
+adding more nodes in a flat distributed cluster just increases the overall system cost without any serving performance gain except increasing overall 
+supported feed throughput which will increase almost linearly with number of nodes. 
+
+Two different <em>DQW/(DQW+SQW)</em> factors are  
+are illustrated in the figures below. The overall query work (TQW) is the same for both cases (10 ms) but the <em>DQW</em> portion of the <em>TQW</em> is different. 
+The throughput (QPS) is 
+a function of the latency (<a href="https://en.wikipedia.org/wiki/Little%27s_law">Little's Law</a>) and the number of cpu cores * nodes. Using 1 node with 24 v-cpu cores and 10 ms service time (<em>TQW</em> we expect to reach close to 2400 QPS at 100% utilization (Unless there are other bottlenecks downstream). 
 </p>
+<figure>
+  <img src="../img/ScalingLatencyFactor0.5.svg" />
+  <img src="../img/ScalingLatencyFactor0.005.svg" />
+</figure>
+In the left figure the overall latency is 10 ms but the dynamic query work (latency) is only 50% and from Amdahl's law it follows that the maximum speedup we can get is 2 regardless how many
```

Comment:
The figures are rendered on top of each other if width of page is low, consider naming them as first and second.